### PR TITLE
add get_schema_path util function

### LINKED ISSUE #324: add user utility function to get the path of a schema
As users want to create/validate ASDF files with weldx schemas, we should provide an easy way to access them.

### PR DESCRIPTION
**Pull request by CagtayFabry**
## Changes
add `get_schema_path`

## Related Issues

Closes #324 

## Checks

- [x] updated CHANGELOG.md
- [ ] updated tests
- [ ] updated doc/
- [ ] update example/tutorial notebooks

### CODE REVIEW

File: weldx/asdf/util.py [hunk 1]
```diff
@@ -18,6 +21,21 @@
     "notebook_fileprinter",
 ]
 
+
+def get_schema_path(schema: str) -> Path:
```
Comment (marscher) [inline weldx/asdf/util.py:25]: could a schema argument include a "path" to get a subtree?
e.g. "single_proc.geometry" or "single_proc.process.wire_feed"

Comment (marscher) [reply weldx/asdf/util.py:25]: eventually this should be a separate function

File: weldx/asdf/util.py [hunk 2]
```diff
@@ -18,6 +21,21 @@
...
+    """Get the path to a weldx schema file."""
+    schema = schema.split(".yaml")[0]
+
+    p = WELDX_PATH / "asdf" / "schemas"
+    print(p)
+    schemas = list(p.glob(f"**/{schema}.yaml"))
```
Comment (marscher) [inline weldx/asdf/util.py:31]: why globbing here, you could just do (p + schema).exists()?

Comment (CagtayFabry) [reply weldx/asdf/util.py:31]: idea is to only require `single_pass_weld-1.0.0.schema` isntead of `weldx/datamodels/single_pass_weld-1.0.0` or other longer path strings

File: weldx/asdf/util.py (hunk 1 above)
Comment (CagtayFabry) [reply weldx/asdf/util.py:25]: yeah this can be beefed up later for sure

**Comment by CagtayFabry** [comment]
So should we just use this for now? @marscher

File: weldx/asdf/util.py (hunk 2 above)
Comment (marscher) [reply weldx/asdf/util.py:42]: I see. Thanks!

**Comment by marscher** [comment]
Yes, I think this is ready to go.

**Review by marscher — APPROVED** [review]

File: weldx/asdf/util.py [hunk 3]
```diff
@@ -18,6 +21,32 @@
...
+    Returns
+    -------
+    Path
+        Path to the requested schema file in the current filesystem.
+
+    """
+    schema = schema.split(".yaml")[0]
+
+    p = WELDX_PATH / "asdf" / "schemas"
+    print(p)
```
Comment (vhirtham) [inline weldx/asdf/util.py:41]: Is this supposed to stay?

**Review by vhirtham — APPROVED** [review]

File: weldx/asdf/util.py (hunk 3 above)
Comment (CagtayFabry) [reply weldx/asdf/util.py:41]: nope 😁

Comment (CagtayFabry) [inline weldx/asdf/util.py:41]:
```suggestion
```

Comment (CagtayFabry) [reply weldx/asdf/util.py:41]:
```suggestion
```

**Pull request merged**